RatRazor/Pages/Overlay/Index.razor

```diff
@@ -16,19 +17,27 @@
         {
             @if (itemScan is ItemIconScan iconScan)
             {
-                <div style="font-size: smaller; position: absolute; left: @(iconScan.GetToolTipPosition().X - VirtualScreenOffset.XOffset)px; top: @(iconScan.GetToolTipPosition().Y - VirtualScreenOffset.YOffset)px; width: @(iconScan.Icon.ItemSize.X)px; height: @(iconScan.Icon.ItemSize.Y)px; ">
+                <div style="font-size: smaller; position: absolute; left: @(GetItemOverlayPosition(itemScan).X + 3)px; top: @(GetItemOverlayPosition(itemScan).Y + 3)px; width: @(iconScan.Icon.ItemSize.X)px; height: @(iconScan.Icon.ItemSize.Y)px; ">
                     <div style="height: 100%; width: 100%; position: relative; text-shadow: 0px 0px 5px black; outline-offset: -1px; outline: 1px solid #555555;" class="pa-1">
                         <div style="position: absolute; left: 0px; top: 0px;">
                             <div style="position: relative;">
                                 @{
                                     var iconStyle = "transform-origin: top left;";
+                                    string iconWidth, iconHeight;
                                     if (iconScan.Rotated)
                                     {
-                                        iconStyle += "transform: rotate(0.25turn) translate(0%, -100%);";
+                                        // Its necessary to explicitly define the width and height before the transform in the style property.
```

Comment:
The current way rotation and size was implemented, icons were being displayed the original rotation all the time.